toml/stringify_test.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { assertEquals, assertThrows } from "../assert/mod.js";
+import { stringify } from "./stringify.js";
```

Comment:
Just to clarify, this is a TypeScript codebase.
```suggestion
import { assertEquals, assertThrows } from "../assert/mod.ts";
import { stringify } from "./stringify.ts";
```